core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -1026,6 +1026,9 @@ class EMConfig {
     @Cfg("Tracking of SQL commands to improve test generation")
     var heuristicsForSQL = true
 
+    @Cfg("Tracking of Mongo commands to improve test generation")
+    var heuristicsForMongo = true
```

Comment:
it should be marked as `@Experimental`, with `false` as default value. only once stable we can put it on by default